### PR TITLE
Adds support for clob properties in Hibernate mapping

### PR DESCRIPTION
**Pull request by davidgarth**
Adds a mapping to support String properties annotated with @Lob. Note sure if this is the best way to add `@Lob` support though.

### CODE REVIEW

**Comment by ryanhall07** [comment]
can you add a test?

File: misk-hibernate/src/main/kotlin/misk/hibernate/SessionFactoryService.kt [hunk 1]
```diff
@@ -192,6 +192,7 @@ internal class SessionFactoryService(
       "char" -> Char::class
       "float" -> Float::class
       "double" -> Double::class
+      "materialized_clob" -> String::class
```
Comment (ryanhall07) [inline misk-hibernate/src/main/kotlin/misk/hibernate/SessionFactoryService.kt]: what if the lob does not have a character type?

Comment (davidgarth) [reply misk-hibernate/src/main/kotlin/misk/hibernate/SessionFactoryService.kt]: Good point, this can't be done here.

**Comment by davidgarth** [comment]
Need to find a better way to do this.

**Pull request closed without merging**